src/cpp/src/visual_language/phi3_vision/classes.hpp

```diff
@@ -40,6 +40,14 @@ class VisionEncoderPhi3V : public VisionEncoder {
         const ov::AnyMap properties);
 
     EncodedImage encode(const ov::Tensor& image, const ov::AnyMap& config_map) override;
+
+private:
+    bool use_ov_image_preprocess = false; // Default to false, will be set based on device and environment
+
+    // GPU preprocessing model creation function
+    std::shared_ptr<ov::Model> create_combined_preprocessing_vision_model(
```

Comment:
Does `create_combined_preprocessing_vision_model()` need to be a member function? Convert to standalone function in anonymous namespace

Reply:
Fixed